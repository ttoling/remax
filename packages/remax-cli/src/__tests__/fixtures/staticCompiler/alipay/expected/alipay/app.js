'use strict';

Object.defineProperty(exports, '__esModule', { value: true });

var _rollupPluginBabelHelpers = require('./_virtual/_rollupPluginBabelHelpers.js');
var React = require('react');

var _App =
/*#__PURE__*/
function (_React$Component) {
  _rollupPluginBabelHelpers.inherits(_App2, _React$Component);

  function _App2() {
    _rollupPluginBabelHelpers.classCallCheck(this, _App2);

    return _rollupPluginBabelHelpers.possibleConstructorReturn(this, _rollupPluginBabelHelpers.getPrototypeOf(_App2).apply(this, arguments));
  }

  _rollupPluginBabelHelpers.createClass(_App2, [{
    key: "render",
    value: function render() {
      return this.props.children;
    }
  }]);

  return _App2;
}(React.Component);

exports.default = _App;
