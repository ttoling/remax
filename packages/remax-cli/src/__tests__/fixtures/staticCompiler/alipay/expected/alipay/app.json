{
  "pages": [
    "pages/index",
    "pages/entry/index"
  ],
  "window": {
    "defaultTitle": "Alipay App",
    "titleBarColor": "#323239"
  }
}